src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -92,7 +110,9 @@ const convertInclude = async includeBlock => {
     type,
     model: {
       href,
-      html: await fetchMarkup(buildIncludeUrl(href, includeType)),
+      html: await fetchMarkup(
+        buildIncludeUrl(href, includeType, restParams[2]),
```

Comment:
```suggestion
        buildIncludeUrl(href, includeType, pathname),
```
If you agree with the suggestion above?

Reply:
Was literally just implementing this haha. Although I'm debating whether getting the third item in the array is the best method to do this. Because when it comes to writing a unit test for this I'm going to have to mock the unused rest params which seems a bit weird. I could grab the last item in the array with `.slice(-1)[0]` but this could break if another parameter is introduced. What do you think?

Reply:
Erm, tbh, I'd consider biting the bullet and refactoring use named parameters everywhere, any solution other than this is pretty brittle. Maybe, go with `const pathname = restParams[2];` then raise a followup PR afterwards with refactor to name parameters. It should be a simple refactor but quite a few files.